_includes/v21.2/sidebar-data/migrate.json

```diff
@@ -0,0 +1,84 @@
+{
+  "title": "Migrate",
```

Comment:
```suggestion
  "title": "Migration",
```

Reply:
Leaving for now as it matches the imperative mood of our other top-level sections.